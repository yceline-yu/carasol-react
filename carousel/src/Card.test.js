import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";

it("renders without crashing", function(){
  render(<Card caption="test Caption" src="./image1.jpg" currNum="1" totalNum="1"/>)
});

it("matches snapshot", function () {
  const { container } = render(<Card caption="test Caption" src="./image1.jpg" currNum="1" totalNum="1"/>);

  expect(container).toMatchSnapshot();
})