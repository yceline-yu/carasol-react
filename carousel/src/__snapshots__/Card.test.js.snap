// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      test Caption
    </h4>
    <img
      alt="test Caption"
      class="Card-image"
      src="./image1.jpg"
    />
    <small
      class="Card-small"
    >
      Image 
      1
       of 
      1
      .
    </small>
  </div>
</div>
`;
